applications/GeoMechanicsApplication/tests/cpp_tests/test_time_stepping.cpp

```diff
@@ -0,0 +1,154 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Wijtze Pieter Kikstra
+//                   Anne van de Graaf
+//
+
+#include "testing/testing.h"
+#include "custom_workflows/time_step_executor.h"
+#include "custom_workflows/strategy_wrapper.hpp"
+#include "solving_strategies/strategies/solving_strategy.h"
+
+using namespace Kratos;
+
+
+namespace
+{
+
+class ProcessSpy : public Process
+{
+public:
+    void ExecuteInitializeSolutionStep() override {mSolutionStepInitializedCalls += 1;}
+    void ExecuteFinalizeSolutionStep()   override {mSolutionStepFinalizedCalls   += 1;}
```

Comment:
Although correct, I think it's more common to use the pre-increment operator (`++`) if you want to increase a variable's value by one:
```suggestion
    void ExecuteInitializeSolutionStep() override {++mSolutionStepInitializedCalls;}
    void ExecuteFinalizeSolutionStep()   override {++mSolutionStepFinalizedCalls;}
```
(What you wrote here is common in Python though, since Python doesn't have increment operators.)

Reply:
Learned something, done.